iocore/net/QUICNetVConnection.cc

```diff
@@ -486,10 +486,11 @@ QUICNetVConnection::handle_frame(QUICEncryptionLevel level, const QUICFrame &fra
     // draining. Receiving a closing frame is sufficient confirmation, as is receiving a stateless reset.
     if (frame.type() == QUICFrameType::APPLICATION_CLOSE) {
       this->_switch_to_draining_state(QUICConnectionErrorUPtr(
-        new QUICConnectionError(QUICErrorClass::APPLICATION, static_cast<const QUICApplicationCloseFrame &>(frame).error_code())));
+        std::make_unique<QUICConnectionError>(static_cast<const QUICApplicationCloseFrame &>(frame).error_code())));
```

Comment:
`QUICErrorClass::APPLICATION` should be used in first argument, otherwise default QUICErrorClass ( `QUICErrorClass::TRANSPORT` ) will be used.

Reply:
NVM, I didn't noticed that you also changed constructor.